src/components/layout/MainLayout.vue

```diff
@@ -0,0 +1,100 @@
+<template>
+    <Layout class="main-layout">
+        <Sider
+            class="main-layout__sider"
+            breakpoint="md"
+            :collapsible="isCurrentMediaQueryTouch"
+            hide-trigger
+            v-model="isCollapsed"
+            v-click-outside="closeSider"
+        >
+            <slot name="sider" />
+        </Sider>
+        <Layout>
+            <Header class="main-layout__header" data-cy="top-navbar">
+                <Icon
+                    class="main-layout__header__menu-icon"
+                    type="md-menu"
+                    size="24"
+                    @click.native="collapseSider"
+                />
+                <slot name="header" />
+            </Header>
+            <Content class="main-layout__content">
+                <slot />
+            </Content>
+        </Layout>
+    </Layout>
+</template>
+
+<script lang="ts">
+import { Component, Vue } from 'vue-property-decorator';
+
+@Component({
+    name: 'MainLayout',
+})
+export default class MainLayout extends Vue {
+    private isCollapsed: boolean = false;
+
+    private collapseSider(): any {
+        this.isCollapsed = !this.isCollapsed;
+    }
+
+    private closeSider(): any {
+        this.isCollapsed = true;
+    }
+
+    public get isCurrentMediaQueryTouch(): boolean {
+        // @ts-ignore
+        return this.$mq === 'sm';
+    }
+}
+</script>
+
+<style lang="less" scoped>
+@import '../../styles/utilities.less';
```

Comment:
@Andresws12 Why import utilities in the component? I would like learn the motive.